src/components/manager.ts

```diff
@@ -87,6 +85,27 @@ export class Manager {
         return path.normalize(out).split(path.sep).join('/')
     }
 
+    /**
+     * @returns the indicator if the root file
+     */
+    getRootIndictor() {
+        const configuration = vscode.workspace.getConfiguration('latex-workshop')
+        const indicator = configuration.get('latex.rootFile.indicator')
+        let ROOT_INDICATOR = null
+        switch (indicator) {
+            case '\\documentclass':
+                ROOT_INDICATOR = /\\documentclass(?:\s*\[.*\])?\s*\{.*\}/ms
+                break
+            case '\\begin{document}':
+                ROOT_INDICATOR = /\\begin{document}/m
+                break
```

Comment:
Do not use a CONSTANT variable in function. For this implementation, returning the corresponding regex is sufficient.

Reply:
<img width="811" alt="image" src="https://user-images.githubusercontent.com/30233673/234466253-cc795d37-40fe-484f-85c2-d41e95a84110.png">

What should I use? Const or let?